src/module/actor-base-sheet.ts

```diff
@@ -43,15 +43,20 @@ export class BaseActorSheet<T extends ActorSheet.Data> extends ActorSheet<
 
   async populateItemList(
     itemType: string,
-    candiateItems: { [name: string]: Item }
+    candiateItems: { [name: string]: Item },
+    itemSubType?: string
   ): Promise<void> {
     for (const e of game.packs) {
       if (
         e.metadata.private == false &&
         ((game?.release?.generation >= 10 && e.metadata.type === "Item") ||
           (game?.release?.generation < 10 && e.metadata.entity === "Item"))
       ) {
-        const items = (await e.getDocuments()).filter(
+        const items = itemSubType ? (await e.getDocuments()).filter(
+          (i) => 
+            (<SWNRBaseItem>i).type == itemType &&
+            (<SWNRBaseItem>i).system.type == itemSubType
```

Comment:
system isn't supported in v9 and right now i am doing all my dev to support v9 for the near future. i will make the change after the merge, but will need to use data.data for the time being.

Reply:
oh damn, missed that